.github/workflows/crowdin-download.yml

```diff
@@ -0,0 +1,36 @@
+name: Crowdin download
+
+on:
+  schedule:
+    - cron: '0 1 * * *' # every day at 1am we download translations from Crowdin and make PR to main
+
+  # manual trigger from Github UI - Action tab
+  workflow_dispatch:
+
+jobs:
+  download-translations:
+    name: Download translations from Crowdin
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Extract translations
+        run: 'yarn i18n:extract'
+
+      - name: Download from Crowdin
+        uses: crowdin/github-action@1.4.4
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_translations: false
+          download_translations: true
+          create_pull_request: true
+          project_id: '492549'
```

Comment:
I know we tried this, but just want to triple check we can't source this from `secrets`. Want to give it another go now that we're running in the upstream repo?

Reply:
I don't see a problem exposing the project ID. It can easily be found in the Crowdin UI publicly (https://crowdin.com/project/juicebox-interface).

Reply:
Agree there's nothing wrong with exposing. But it would be nice to have a single source of truth for the value, rather than it being defined twice in both workflow files.

But totally not a big deal 👍

Reply:
Ahh yep that is a good point.

Reply:
I don't see a problem exposing the project ID. It can be found easily in the Crowdin UI publicly anyway (https://crowdin.com/project/juicebox-interface). Also, Uniswap hard codes it into their action files like this, so I don't think it's be a problem. 

> @johnnyd-eth the benefit to both is that it creates a single source of truth for the config. Currently to make any edits to the Crowdin behavior, we have to edit 2 files.
> 
> I don't think this is a problem, though, hence non-blocking - there's only 2 files, the likelihood of needing to change the config is low, etc. Just wanted to note it for potential future improvements :)

Sounds good 👌